### ISSUE TITLE
glenn2vcf slow on mhc

### ISSUE THREAD

**Issue by glennhickey** [BD2KGenomics/hgvm#423]
glenn2vcf seems to take forever (I had some go all night last night) on some mhc graphs.  Hopefully there's something that can be done to speed it up? 

Example command line that's been running for over an hour

 glenn2vcf /cluster/home/hickey/ga4gh/hgvm-graph-bakeoff-evalutations/paper_v5.8/primary_02_i_0.90_delt_0.05_ss_10000/mhc/prg/NA12878_augmented.vg /cluster/home/hickey/ga4gh/hgvm-graph-bakeoff-evalutations/paper_v5.8/primary_02_i_0.90_delt_0.05_ss_10000/mhc/prg/NA12878_call.tsv -o 28510119 -r ref -c 6 -s NA12878

**Comment by glennhickey** [comment]
Actually, replacing "prg" by "cactus" in the command line below gives a longer running job (150 min and counting), for whatever that's worth

**Comment by adamnovak** [comment]
OK, I have fixed this. I had a max search depth of 100 for my path-to-the-reference search, and as I had to throw out some of the ordinary BFS restrictions (because we want longest paths and not shortest paths), it was generating way too many search intermediates.

I've knocked the depth limit down to 10 nodes, and made it controllable with the -d option. Lowering it makes more nodes be dropped as having no way to get to the primary path, but now the MHC run finishes in something like 30 seconds.

Try the new version I have pushed up.

The long-term right answer might be a proper longest-path or path-picking-up-the-most-node-copy-number algorithm, instead of a depth-limited brute-force BFS.
